foreman/foreman.spec

```diff
@@ -855,6 +855,9 @@ rm -rf %{buildroot}
 %{_datadir}/%{name}/plugins
 %{_datadir}/%{name}/public
 %{_datadir}/%{name}/Rakefile
+%exclude %{_datadir}/%{name}/script/%{name}-tail.d/common
```

Comment:
Instead of blacklisting named dirs here (which will cause the bug to recur when dirs are added), delete any dirs beneath `%{name}-tail.d/` in the `%install` section.